app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -37,6 +37,7 @@ public enum SettingsEnum {
     ADREMOVER_FEED_SURVEY_REMOVAL("revanced_adremover_feed_survey", true, ReturnType.BOOLEAN),
     ADREMOVER_SHORTS_REMOVAL("revanced_adremover_shorts", true, ReturnType.BOOLEAN, true),
     ADREMOVER_COMMUNITY_GUIDELINES_REMOVAL("revanced_adremover_community_guidelines", true, ReturnType.BOOLEAN),
+    ADREMOVER_SUBSCRIBERS_COMMUNITY_GUIDELINES_REMOVAL("revanced_adremover_subscribers_community_guidelines_removal", false, ReturnType.BOOLEAN),
```

Comment:
This should be true by default. Don't forget to reflect the changes in the patches as well.